Buildeploy.targets

```diff
@@ -1,408 +1,417 @@
-    <Error Condition="$(NugetExe)==''" Text="Property 'NugetExe' is not specified."/>
-    <Error Condition="$(VersionRedistRoot)==''" Text="Property 'VersionRedistRoot' is not specified."/>
-    <Error Condition="$(Version)==''" Text="Property 'Version' is not specified."/>
-    <Error Condition="%(OctoConfig.SkipClickOnce)!=true and $(ClickOnceCertificate)==''" Text="Property 'ClickOnceCertificate' is not specified."/>
-    <Error Condition="%(OctoConfig.SkipClickOnce)!=true and $(ClickOnceEntryPoint)==''" Text="Property 'ClickOnceEntryPoint' is not specified."/>
-    <Error Condition="%(OctoConfig.SkipClickOnce)!=true and $(ProductName)==''" Text="Property 'ProductName' is not specified."/>
-    <Error Condition="%(OctoConfig.SkipClickOnce)!=true and $(Publisher)==''" Text="Property 'Publisher' is not specified."/>
-
-
-    <ItemGroup Condition="@(ClientRedistFiles)==''">
-      <ClientRedistFiles Include="$(TempClientRedist)\**"/>
-    </ItemGroup>
-
-    <ItemGroup>
-      <CompiledSiteWithConfig Include="@(CompiledSite)"/>
-      <CompiledSiteWithConfig Include="$(WebsiteSource)\Web*.config"/>
-    </ItemGroup>
-
-    <NugetPackage
-            ConfigName="%(OctoConfig.Identity)"
-            NuspecFile="%(OctoConfig.NuspecFile)"
-            NugetExePath="$(NugetExe)"
-            OutputDirectory="$(PackagesDirPath)"
-            UseConfigName="$(ClickOnceUseConfigName)"
-            AutoUpdate="%(OctoConfig.AutoUpdate)"
-            BasePath="$(TempClientRedist)"
-            Certificate="$(ClickOnceCertificate)"
-            CertificatePassword="$(ClickOnceCertificatePassword)"
-            ConfigFile="%(OctoConfig.ConfigFile)"
-            EntryPoint="$(ClickOnceEntryPoint)"
-            Files="@(ClientRedistFiles)"
-            Install="%(OctoConfig.Install)"
-            LocalProduct="%(OctoConfig.ProductName)"
-            Product="$(ProductName)"
-            Publisher="$(Publisher)"
-            Url="%(OctoConfig.Url)"
-            UrlParameters="$(ClickOnceUrlParameters)"
-            MinimumRequiredVersion="%(OctoConfig.MinimumRequiredVersion)"
-            RequireLatestVersion="%(OctoConfig.RequireLatestVersion)"
-            Version="$(Version)"
-            TargetFramework="$(ClickOnceTargetFramework)"
-            CreateDesktopShortcut="%(OctoConfig.CreateDesktopShortcut)"
-            IconFile="%(OctoConfig.IconFile)"
-            WebsiteBasePath="$(CompiledSite)"
-            CombineWithWebsite="%(OctoConfig.CombineWithWebsite)"
-            WebsiteFiles="@(CompiledSiteWithConfig)"
-			      LinkAssembliesWithManifestAsFile="$(ClickOnceLinkAssembliesWithManifestAsFile)"
-            SkipClickOnce="%(OctoConfig.SkipClickOnce)"
-            PackageId="%(OctoConfig.PackageId)"
-            PushLocation="%(OctoConfig.PushLocation)"
-            ApiKey="%(OctoConfig.ApiKey)"
-            />
-    <Copy SourceFiles="$(VersionRedistRoot)\ClickOnce\$(ClickOnceFilePrefix)%(OctoConfig.Identity)_$(Version).zip"

```

Comment:
OctoPackage is not related to ClickOnce.

On Sat, Jun 20, 2015 at 7:46 PM Alexander Powolozki <
notifications@github.com> wrote:

> In Buildeploy.targets
> <https://github.com/CMSSudhaus/buildeploy/pull/1#discussion_r32886411>:
>
> > -            MinimumRequiredVersion="%(OctoConfig.MinimumRequiredVersion)"
> > -            RequireLatestVersion="%(OctoConfig.RequireLatestVersion)"
> > -            Version="$(Version)"
> > -            TargetFramework="$(ClickOnceTargetFramework)"
> > -            CreateDesktopShortcut="%(OctoConfig.CreateDesktopShortcut)"
> > -            IconFile="%(OctoConfig.IconFile)"
> > -            WebsiteBasePath="$(CompiledSite)"
> > -            CombineWithWebsite="%(OctoConfig.CombineWithWebsite)"
> > -            WebsiteFiles="@(CompiledSiteWithConfig)"
> > -			      LinkAssembliesWithManifestAsFile="$(ClickOnceLinkAssembliesWithManifestAsFile)"
> > -            SkipClickOnce="%(OctoConfig.SkipClickOnce)"
> > -            PackageId="%(OctoConfig.PackageId)"
> > -            PushLocation="%(OctoConfig.PushLocation)"
> > -            ApiKey="%(OctoConfig.ApiKey)"
> > -            />
> > -    <Copy SourceFiles="$(VersionRedistRoot)\ClickOnce\$(ClickOnceFilePrefix)%(OctoConfig.Identity)_$(Version).zip"
>
> Deleted copy commands are in my opinion copy / pasted commands which are
> exactly the same as ised during ClickOnce job. That's why i thought, their
> presence in the OctoPackage Target is a mistake, causing files to be copied
> two times, if they would be generated.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CMSSudhaus/buildeploy/pull/1/files#r32886411>.
>